src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js

```diff
@@ -376,18 +368,30 @@ export const NewAnnotationForm = ({
 								showConditionRelations={showConditionRelations}
 								errorMessages={exConErrorMessages}
 								searchService={searchService}
+								buttonIsDisabled={(
+									newAnnotation.conditionRelations[0]
+									&& newAnnotation.conditionRelations[0].handle
+								)}
 							/>
 						</SplitterPanel>
 						<SplitterPanel style={{paddingRight: '10px', paddingTop: '6vh'}} size={30}>
 							<label htmlFor="experiments">Experiments</label>
 							<ConditionRelationHandleFormDropdown
 								name="experiments"
+								customRef={experimentsRef}
 								editorChange={onDropdownExperimentsFieldChange}
 								referenceCurie={newAnnotation.singleReference.curie}
 								value={newAnnotation.conditionRelations[0]?.handle}
 								showClear={false}
 								placeholderText={newAnnotation.conditionRelations[0]?.handle}
-								isEnabled={isExperimentsEnabled}
+								isEnabled={
+									(
```

Comment:
@adamgibs wouldn't this be better to call a method vs inline code?

Reply:
@oblodgett Sure, I just updated it with the logic in functions